src/app/introduction/introduction.component.scss

```diff
@@ -0,0 +1,145 @@
+@import "../styles/components/_variables";
+
+::selection {
+    background: $yellow; /* WebKit/Blink Browsers */
+}
+::-moz-selection {
+    background: $yellow; /* Gecko Browsers */
+}
+h3 {
+    text-align: center;
+    font-weight: $font-bold;
+    color: $blue;
+}
+h2 {
+    font-weight: $font-bold;
+}
+
+.mosquito {
+    width: 80px;
+    height: auto;
+}
+
+.lead-title {
+    font-weight: $font-bold;
+    font-size: 2.5em;
+    color: $black;
+}
+
+.lead-sub-heading {
+    color: grey;
+    font-weight: $font-bold;
+    font-size: 1.5em;
+}
+
+@mixin border-radius($radius1, $radius2) {
+    -webkit-border-bottom-right-radius: $radius1 $radius2;
+    -moz-border-radius-bottomright: $radius1 $radius1;
+    border-bottom-right-radius: $radius1 $radius1;
+}
+
+.rectangle-speech-border {
+  position: relative;
+  padding: 50px;
+  width: 100%;
+  margin: 1em 0 3em;
+  border: 2px solid black;
+  color: #333;
+  background: $white;
+  -webkit-border-radius: 50px;
+  -moz-border-radius: 50px;
+  border-radius: 50px;
+  &:before,
+  &:after {
+    content: "";
+    position: absolute;
+    z-index: 10;
+    bottom: -30px;
+    height: 30px;
+    border-style: solid;
+    border-width: 0 2px 2px 0;
+    border-color: $black;
+    background: transparent;
+    display: block;
+  }
+  &:before {
+    left: 50px;
+    width: 50px;
+    @include border-radius(80px, 50px);
+  }
+  &:after {
+    left: 55px;
+    width: 20px;
+    @include border-radius(40px, 50px);
+  }
+  > :first-child:after {
+    content: "";
+    position: absolute;
+    bottom: -10px;
+    left: 76px;
+    width: 24px;
+    height: 15px;
+    background: $white;
+  }
+}
+
+.arrow {
+    width: 75px;
+    line-height: 75px;
+    text-align: center;
+    opacity: 1;
+    border: none;
+    color: $white;
+    border-radius: 50%;
+}
+
+.arrow-red {
+    @extend .arrow;
+    background: #d9534f;
+}
+
+.arrow-green {
+    @extend .arrow;
+    background: green;    
+}
+
+@mixin animation($animate) {
+    -moz-animation: $animate;
+    -webkit-animation: $animate;
+    animation: $animate;
+}
+
+@mixin keyframes($animationName) {
+    @-webkit-keyframes #{$animationName} {
+        @content;
+    }
+    @-moz-keyframes #{$animationName} {
+        @content;
+    }
+    @-o-keyframes #{$animationName} {
+        @content;
+    }
+    @keyframes #{$animationName} {
+        @content;
+    }
+}
+
+.bounce {
+    @include animation(bounce 2s infinite)
+}
+
+@include keyframes(bounce) {
+    0%, 20%, 50%, 80%, 100% {
+        transform: translateY(0);
+    }
+    40% {
+        transform: translateY(-30px);
+    }
+    60% {
+        transform: translateY(-15px);
+    }
+}
+
+#file {
+    display: none;
```

Comment:
`display: none` is actually a [class](http://getbootstrap.com/css/#responsive-utilities-classes) in bootstrap